app/services/album.js

```diff
@@ -0,0 +1,15 @@
+const rp = require('request-promise');
+const { albums_api } = require('../../config').api_services;
+const { defaultError } = require('../errors');
+
+const getAllAlbums = () =>
+  rp(`${albums_api}albums`)
+    .then(res => JSON.parse(res))
+    .catch(err => Promise.reject(defaultError(err.message)));
+
+const getPhotosAlbum = id =>
+  rp(`${albums_api}albums/${id}/photos`)
+    .then(res => JSON.parse(res))
```

Comment:
As i comment in previous review, request-promise has an [option](https://github.com/request/request-promise#get-something-from-a-json-rest-api) for parse a json response without this.